docker/run.sh

```diff
@@ -6,16 +6,18 @@ then
     exit 1
 fi
 
+rm ./wiremock-data/grpc/services.dsc
+
 protoc \
   --descriptor_set_out wiremock-data/grpc/services.dsc \
-  --proto_path proto_files \
-  proto_files/wallet/myservice.proto
+  --proto_path proto_files/wallet \
```

Comment:
With such change it may also work.
But in practise in our repository proto files has folder hierarchy:

proto_files
-> wallet
    -> file1.proto
    -> file2.proto
-> common
     -> commonfile1.proto

And such structure doesn't work